### PR TITLE
Switch to Azure Pipelines

### PR DESCRIPTION
**Pull request by nickw444**
Travis CI (.org) has been placed into readonly mode, forcing a manual migration to Travis CI (.com). I've since been migrating my projects over to Azure, and now the time has come for Flask LDAP3 Login. 

Happy to keep this on Travis CI (.com), but I trust Microsoft to be a bit more supportive of OSS moving forward.

### CODE REVIEW

**Review by gmacon — APPROVED** [review]
LGTM

**Comment by nickw444** [comment]
@gmacon once merged, I'd like to test the automatic tag release function in Azure - Are you OK with me releasing an alpha version (a1) of the next minor semver version? I'll delete the tag and  release from PyPi once proven successful.

**Comment by gmacon** [comment]
That sounds fine to me.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          tox.env: 'py35'
+        Python36:
+          python.version: '3.6'
+          tox.env: 'py36'
+        Python37:
+          python.version: '3.7'
+          tox.env: 'py37'
+        Python38:
+          python.version: '3.8'
+          tox.env: 'py38'
```
Comment (HeMan) [inline azure-pipelines.yml:72]: Should perhaps Python 3.9 be in the matrix?

Comment (nickw444) [reply azure-pipelines.yml:72]: My intent was to maintain parity with the existing CI setup, and add it later... but it's a good point, I'll add it in this PR if it "just works".

**Comment by nickw444** [comment]
Thanks @gmacon for merging. Good for the interim, however I actually discovered we might be able to use Github actions instead, as a more "all in one" setup.

**Pull request merged**